src/ui/templates/controls/filteroptions.hbs

```diff
@@ -28,25 +28,26 @@
       {{/if}}
       {{#if showReset}}
```

Comment:
Do we sill need this outer showReset here and below? We're not conditionally rendering the button any longer, we're just triggering display:none via displayReset

Reply:
Oh I'm not sure! There are two cases where the reset button doesn't appear: 

(1) `showReset: true` but no options are selected. This is where we toggle display: none 

(2) implementer specifies `showReset: false` in their config, in which case the reset button wouldn't appear in the DOM at all

Should we change case (2) so that the reset button does appear in the DOM but is hidden?

Reply:
Yeah, let's update case (2) so that the button is always there. I think it will simplify the code a bit, we won't have to track the difference between displayResult and showResult (which I obviously can't do, haha)

Reply:
Hahaha, ok will update!